src/Classes/Database.php

```diff
@@ -105,7 +105,7 @@ public function query($sql, $params = [], $rollback = false)
             }
         }
 
-        if (isset($_REQUEST['dbdbg']) && CoreUtils::hasPermission(AUTH_SHOWERRORS)) {
+        if (isset($_REQUEST['dbdbg']) && AuthUtils::hasPermission(AUTH_SHOWERRORS)) {
```

Comment:
Heh, actually I want to deprecate hasPermission to explicitly use the User class at some point, but that's good for now.